specification/automation/resource-manager/Microsoft.Automation/stable/2018-06-30/python2package.json

```diff
@@ -0,0 +1,733 @@
+          "description": "Gets or sets the error message."
+        }
+      },
+      "description": "Definition of the module error info type."
+    },
+    "ModuleListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Module"
+          },
+          "description": "Gets or sets a list of modules."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets or sets the next link."
+        }
+      },
+      "description": "The response model for the list module operation."
+    },
+    "ContentHash": {
+      "properties": {
+        "algorithm": {
+          "type": "string",
+          "description": "Gets or sets the content hash algorithm used to hash the content."
+        },
+        "value": {
+          "type": "string",
+          "description": "Gets or sets expected hash value of the content."
+        }
+      },
+      "required": [
+        "algorithm",
+        "value"
+      ],
+      "description": "Definition of the runbook property type."
+    },
+    "ContentLink": {
+      "properties": {
+        "uri": {
+          "type": "string",
+          "description": "Gets or sets the uri of the runbook content."
+        },
+        "contentHash": {
+          "$ref": "#/definitions/ContentHash",
+          "description": "Gets or sets the hash."
+        },
+        "version": {
+          "type": "string",
+          "description": "Gets or sets the version of the content."
+        }
+      },
+      "description": "Definition of the content link."
+    },
+    "ModuleCreateOrUpdateProperties": {
+      "properties": {
+        "contentLink": {
+          "$ref": "#/definitions/ContentLink",
+          "description": "Gets or sets the module content link."
+        }
+      },
+      "required": [
+        "contentLink"
+      ],
+      "description": "The parameters supplied to the create or update module properties."
+    },
+    "ModuleCreateOrUpdateParameters": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ModuleCreateOrUpdateProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the module create properties."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets name of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets the tags attached to the resource."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "The parameters supplied to the create or update module operation."
+    },
+    "ModuleProperties": {
+      "properties": {
+        "isGlobal": {
```

Comment:
bools are typically not recommended as they are less descriptive and dont allow for future expansion of states if needed.

Reply:
The property isGlobal is set in out data model as a boolean everywhere. Its not expected to change ever. The specs for Python 2 packages are practically the same a Powershell modules for consistency, except for paths.